src/components/ProcessCreate/Questions/useVotingType.ts

```diff
@@ -16,7 +16,6 @@ export const VotingTypes = [VotingTypeSingle as VotingType, UnimplementedVotingT
 export const useVotingType = (): GenericFeatureObject<VotingType> => {
   const { t } = useTranslation()
   return {
-    list: VotingTypes,
     defined: import.meta.env.features.voting_type as VotingType[],
```

Comment:
I am agree with that but this affect the legacy scaffold code. We should rename it there

Reply:
`TypeSelector` has inherited its name from an old implementation which contained a component named `TypeSelector`. It certainly should be called `useCensusTypes` now, since it only contains a hook named that way.

Reply:
but rename it from develop or on d/saas?

Reply:
From develop ofc